src/locales/en/common.json

```diff
@@ -693,6 +701,55 @@
     }
   },
 
+  "walletMigrationArchanova": {
+    "banner": {
+      "title": "Migrate assets from Pillar v1 to Etherspot (Pillar v2)"
```

Comment:
just a note to check re `v2` I think it's marketed as `V2`

Reply:
I've got that Dmitry's designs (e.g. https://app.zeplin.io/project/5fa95ca539e6804a10dec0a4/screen/608befab4cf4eb02d439bb11). Will check with him

Reply:
Dmitry confirmed it's `v2` and not `V2`.

Reply:
All articles, vids, banners, emails use `V2` so I guess it's `V2` 🤔